### ISSUE TITLE
User tokens masking?

### ISSUE THREAD

**Issue by bilunsun** [neuralmagic/AutoFP8#41]
## Question
Does AutoFP8 differentiate between user/input tokens and assistant/output tokens?

## Context
I'm looking at the `example_dataset.py`, specifically this line
```python
examples_encoded = tokenizer(examples, padding=True, truncation=True, return_tensors="pt")
```

I don't see any masking that differentiates between user/input tokens and assistant/output tokens. These are the attributes of the encodings:
```python
Encoding(num_tokens=5013, attributes=[ids, type_ids, tokens, offsets, attention_mask, special_tokens_mask, overflowing])
```

**Comment by mgoin** [comment]
No, it will calibrate based on all tokens provided to the model forward pass

**Comment by bilunsun** [comment]
Thank you for clarifying!
